README.md

```diff
@@ -166,4 +178,22 @@ Outputs a message to the console in mOcKiNgCaSe.
 
 <hr>
 
+<a name="mOcKiNgCaSe.overrideConsole"></a>
+
+### mOcKiNgCaSe.overrideConsole([options])
+Overrides the console.log input annd prints it in the mOcKiNgCaSe.
+
+**Kind**: static method of [<code>mOcKiNgCaSe</code>](#mOcKiNgCaSe)
```

Comment:
```suggestion
**Kind**: static method of [<code>mOcKiNgCaSe</code>](#mOcKiNgCaSe)
**Returns**: mOcKiNgCaSe with default options
```

Reply:
It's actually nice. I will improve my coding standards too.